src/Test/Runner.elm

```diff
@@ -147,3 +156,46 @@ formatLabels formatDescription formatTest labels =
                 |> List.map formatDescription
                 |> (::) (formatTest test)
                 |> List.reverse
+
+
+{-| A `Shrinkable a` is an opaque type that allows you to obtain a value of type
+`a` that is smaller than the one you've previously obtained.
+-}
+type Shrinkable a
+    = S { down : LazyList (RoseTree a), over : LazyList (RoseTree a) }
```

Comment:
:art: I generally prefer to do this:

```
type alias Shrunk a =
  { down : LazyList (RoseTree a)
  , over : LazyList (RoseTree a)
  }

type Shrinkable a =
  Shrinkable (Shrunk a)
```

That way, it's easier to write functions that dealt with the shrunk type only